src/AElf.Kernel.Consensus.AEDPoS/Application/SecretSharingInformationLogEventProcessor.cs

```diff
@@ -34,9 +35,16 @@ public LogEvent InterestedEvent
             }
         }
 
-        public Task ProcessAsync(Block block, TransactionResult result, LogEvent logEvent)
+        public Task ProcessAsync(Block block, Dictionary<TransactionResult, List<LogEvent>> logEventsMap)
         {
-            _secretSharingService.AddSharingInformationAsync(logEvent);
+            foreach (var logEvents in logEventsMap.Values)
+            {
+                foreach (var logEvent in logEvents)
+                {
+                    _secretSharingService.AddSharingInformationAsync(logEvent);
```

Comment:
_secretSharingService.AddSharingInformationAsync should not accept logEvent as parameters